README.md

```diff
@@ -153,6 +153,11 @@ NOTE: This component uses the `slotchange` event and so will not work if you req
 
 <img src="/images/d2l-applied-filters.png?raw=true" width="450">
 
+#### Attributes
+
+- `for`: The id of the `d2l-filter-dropdown` you want to track.
+- `applied-filters-label-text`: (optional) The text displayed in this component's label. Defaults to "Applied Filters:".
```

Comment:
Minor: We don't need to worry about scoping our attribute names the same way we do with `id`s and `class`es.  This would be fine as just `label-text`